lib/config/package.js

```diff
@@ -212,17 +212,13 @@ PackageConfig.prototype.setPrefix = function(jspmPrefix) {
 
     this.file.remove(['jspm']);
 
-    this.depsPrefixed = false;
     this.file.changed = true;
   }
   else if (!this.jspmPrefix && jspmPrefix) {
-    this.jspmPrefix = true;
-    this.depsBase = ['jspm'];
```

Comment:
No, the assignments are unneeded:

- `this.jspmPrefix`  is set [here](https://github.com/sidloki/jspm-cli/blob/a58db636e0a96027c9af0aca229a9ed4016d23db/lib/config/package.js#L221) in the same function
- `this.depsBase` is not used anywhere. In the write and init function `depsBase` is generate from `this.depsPrefix` (see [here](https://github.com/sidloki/jspm-cli/blob/a58db636e0a96027c9af0aca229a9ed4016d23db/lib/config/package.js#L262) and [here](https://github.com/sidloki/jspm-cli/blob/a58db636e0a96027c9af0aca229a9ed4016d23db/lib/config/package.js#L142))